firmware/quadruna/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 set(QUADRUNA_DIR "${FIRMWARE_DIR}/quadruna")
 set(SHARED_QUADRUNA_DIR "${QUADRUNA_DIR}/shared")
 set(SHARED_EMBEDDED_QUADRUNA_DIR "${SHARED_QUADRUNA_DIR}/src")
-
+set(SHARED_CAR "quadruna")
```

Comment:
we want to be able to build for multiple cars at the same time. maybe pass this information as an argument?

Reply:
I am not sure is it correct of not, but if we have a different car and have a CMakeLists for it `firmware/{anotherCar}/CMakeList.txt` with a statement `set(SHARED_QUADRUNA_DIR, "anotherCar")` , then the `${SHARED_QUADRUNA_DIR}` in this function will be `"anotherCar"`; so I guess it will not prevent building multiple cars at same time? I observed that building dev boards will give empty value for `${SHARED_QUADRUNA_DIR}`.

Reply:
oh that is probably something we should be doing more, you're right.